partcad/src/partcad/context.py

```
@@ -716,60 +615,63 @@
+        prj = self.get_project(project_name)
+        if prj is None:
+            pc_logging.error("Package %s not found" % project_name)
+            pc_logging.error("Packages found: %s" % str(self.projects))
+            return {}
+        pc_logging.debug("Retrieving suppliers from %s" % project_name)
+
+        part_suppliers = prj.get_suppliers()
+        if len(part_suppliers) == 0:
+            pc_logging.error("No suppliers found for %s in %s" % (part_name, project_name))
+            return {}
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)